config/Dockerfiles/linchpin/Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM fedora:latest
+LABEL name="contrainfra/linchpin-executor" \
+      maintainer="https://github.com/CentOS-PaaS-SIG/linchpin" \
+      version="0.0.5" \
+      description="LinchPin executable container running Fedora 28"
+
+ENV APP_ROOT=/linchpin/
+ENV PATH=${APP_ROOT}/bin:${PATH} HOME=${APP_ROOT}
+
+COPY conf/ansible.cfg /etc/ansible/ansible.cfg
+COPY conf/linchpin.conf /etc/linchpin.conf
+COPY bin/ ${APP_ROOT}/bin/
+
+RUN curl -L -o /etc/yum.repos.d/beaker-client.repo \
+    https://beaker-project.org/yum/beaker-client-Fedora.repo; \
+    dnf install -y beaker-client \
+        python-pip \
+        ansible \
+        curl \
+        python-devel \
+        python2-shade \
+        openssl-devel \
+        redhat-rpm-config file \
+        net-tools \
+        python-krbV \
+        libxslt-python \
+        libxml2-python \
+        krb5-workstation \
+        libselinux-python \
+        && dnf clean all; \
+        pip install -U pip; \
+        pip install -U setuptools; \
+        pip install linchpin; \
```

Comment:
I know you query the version out of the container later but this ties us to only release containers of what has been pushed to pypi.  What if you want to build a rc or pre-release container?  I think this should install from the git checkout that the jenkinsfile did.

Reply:
We could easily enough put in an environment variable here that allows you to use the `git+ssh://github.com...` syntax, leaving `linchpin` as default. Would that be acceptable? It would also allow jenkins to not use the the 'latest' tag option which is currently provided.

I personally think this would give us the most flexibility because my POV on this was that we'd be doing the pypi release first, then pulling it down, testing it, and running the container build. I don't know that we want 'official' builds from git like that.